activesupport/lib/active_support/core_ext/object/to_query.rb

```diff
@@ -70,14 +78,14 @@ class Hash
   #   {name: 'David', nationality: 'Danish'}.to_query('user')
   #   # => "user%5Bname%5D=David&user%5Bnationality%5D=Danish"
   #
+  # If `escape: false` is provided values are not escaped.
+  #
   # The string pairs "key=value" that conform the query string
   # are sorted lexicographically in ascending order.
-  #
-  # This method is also aliased as +to_param+.
-  def to_query(namespace = nil)
+  def to_query(namespace = nil, escape: true)
```

Comment:
Adding the keyword argument breaks clients which relies on "*args" delegation. I'm not sure how big of an impact this will be but the fix on the client is quite simple.